js/components/date.js

```diff
@@ -1,4 +1,4 @@
-var DATE_FORMAT = 'YYYY-MM-DD';
+var DATE_FORMAT = 'MM-DD-YYYY';
```

Comment:
This is meant to be `dd/mm/yyyy` if you read the linked issue, right?

Reply:
@squallstar 
Actually I thought that both 'MM-DD-YYYY' and 'DD-MM-YYYY' are fine because of this comment.
![Capture](https://user-images.githubusercontent.com/52824207/82950780-97e95a80-9fae-11ea-9a00-1c159b8d8d41.PNG)

Should I remake it to 'DD-MM-YYYY' ?

Reply:
I will discuss with @tonytlwu in the morning to ensure we have considered all applicable scenarios for existing apps and clients in both regions.